xcauto/functionals/lyp.py

```diff
@@ -0,0 +1,55 @@
+# The functional definition in this file was ported to Python
+# from XCFun, which is Copyright Ulf Ekström and contributors 2009-2020
+# and provided under the Mozilla Public License (v2.0)
+# see also:
+#   - https://github.com/dftlibs/xcfun
+#   - https://github.com/dftlibs/xcfun/blob/master/LICENSE.md
+
+
+import jax.numpy as np
+
+
+def lyp_a_b_gaa_gab_gbb(a, b, gaa, gab, gbb):
+    # gab not used
```

Comment:
It is used though.
```suggestion
```

Reply:
Good point, I found that out after I added that comment :-)